wynnset/tapestry-wp
# 594 - Allow creation of tapestry using a shortcode / button / widget

Closes #594 

Adds the ability to create a button on any post via shortcode` new_tapestry_button`.

Clicking the button will prompt for a new tapestry name; if tapestry is created successfully, it will redirect to the new page of the tapestry; otherwise, will alert with an error.

This also changes the add tapestry endpoint object to have a new "permalink" property under the settings object.

HOW TO TEST:
Make a new post with any title, and in the body write `[new_tapestry_button]`. Navigate to the post, and a button should appear to add a new tapestry.
